chall01/jnovotny.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/python3
+import sys
+
+def usage_exit():
+	print("usage: ./jnovotny.py <a-zA-Z string>")
+	exit()
+
+def morse(string):
+	morse_dict = {
+		"a":".-",
+		"b":"-...",
+		"c":"-.-.", 
+		"d":"-..",
+		"e":".",
+		"f":"..-.", 
+		"g":"--.",
+		"h":"....",
+		"i":"..",
+		"j":".---",
+		"k":"-.-",
+		"l":".-..",
+		"m":"--",
+		"n":"-.",
+		"o":"---",
+		"p":".--.",
+		"q":"--.-",
+		"r":".-.",
+		"s":"...",
+		"t":"-",
+		"u":"..-",
+		"v":"...-",
+		"w":".--",
+		"x":"-..-",
+		"y":"-.--",
+		"z":"--.."}
+	str_out = ""
+	for letter in string:
+		if letter == " ":
+			str_out += " "
+		elif letter in morse_dict.keys():
```

Comment:
I would suggest adding space into your dict and reformat this with
```
str_out += morse_dict[letter]
```
and put the `else` out of here as an error case handling before going inside this loop.